### ISSUE TITLE
no docs are shown

### ISSUE THREAD

**Issue by sagiegurari** [inch-ci/inch_ci-web#93]
For some reason it can't find the jsdocs for my new repo: sagiegurari/multiple-redis

Re: [sagiegurari/multiple-redis](https://inch-ci.org/github/sagiegurari/multiple-redis)

**Comment by sagiegurari** [comment]
any updates on this one?
the doc workers seem to be stuck.

**Comment by rrrene** [comment]
**D'oh!** I typed the following two days ago and left the tab open without submitting. Sorry!

---

Clearly a bug in the CI system. If we run the CLI client, we get the following result:

    $ inchjs list

    # Seems really good                                                             
    ┃  A  →  MultiRedisClient.createClient in /lib/multiple-redis.js
    ┃  A  →  MultiRedisClient.createClient in /lib/multiple-redis.js
    ┃  A  →  MultiRedisClient in /lib/multiple-redis.js
    ┃  A  ↓  MultiRedisClient in /lib/multiple-redis.js

    $ inchjs suggest

    Nothing to suggest. Try --pedantic to be excessively concerned with minor details and rules.

But Inch CI somehow is stuck in the evaluation worker. Will try to figure this out this weekend! :+1:

**Comment by sagiegurari** [comment]
Thanks a lot

**Comment by rrrene** [comment]
**TL;DR** I will have to release a version of InchJS that supports multiple signatures per function.

Okay, I figured it out, but this will take a bit of work on my part.

The "problem" is this line: https://github.com/sagiegurari/multiple-redis/blob/a79387a6dd0b919ddca24663ac61c548f1130ed7/lib/multiple-redis.js#L238

InchJS does not (yet) know how to handle multiple signatures per function. JSDoc (which is used for parsing the docs) creates two functions in its output for `MultiRedisClient.createClient` which in turn freaks out the CI service since there can't be two functions with the same identifier inside the same project.

**Comment by sagiegurari** [comment]
ok got you.
basically that's the jsdoc way of documenting overloaded functions.
surprised it hasn't shown up on any other repo until now.

**Comment by sagiegurari** [comment]
any updates on this one?

**Comment by rrrene** [comment]
Yep, it's done. Sorry this took so long: https://inch-ci.org/github/sagiegurari/multiple-redis

**Comment by sagiegurari** [comment]
Thanks a lot
